examples/dt/nova/nova02beta/edpm/nodeset2/values.yaml

```diff
@@ -0,0 +1,165 @@
+# yamllint disable rule:line-length
+# local-config: referenced, but not emitted by kustomize
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: edpm-nodeset-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  root_password: cmVkaGF0Cg==
+  preProvisioned: false
```

Comment:
Assuming this DT should support BMO, I wonder does it miss baremetal specific data and kustomization replacements for it like https://github.com/openstack-k8s-operators/architecture/blob/db92ee55b619bd9824dba5b0334d76a79e6d88c4/examples/dt/bmo01/dataplane/nodesets/values.yaml#L210 does provide?